solutions/devsprint-brq-testes-ios-1/FinanceAppTests/StringExtensionsTests.swift

```diff
@@ -9,5 +9,13 @@ import XCTest
 @testable import FinanceApp
 
 class StringExtensionsTests: XCTestCase {
```

Comment:
Podemos deixar essa classe como final? 

```suggestion
final class StringExtensionsTests: XCTestCase {
```